gamemodes/horde/gamemode/cl_stencil_rings.lua

```diff
@@ -0,0 +1,72 @@
+local colmask = Color(0,0,0,0)
+local render_DrawSphere = render.DrawSphere
+
+rings = rings or {}
+
+local List = {}
+local col = Color(255,255,255,255)
+
+function rings.SetColor(color)
```

Comment:
considering setcolor doesnt really work for multiple colors right now, leave a comment or so, so we at least know about it in the future